index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hein Thant</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="logo">Hein Thant</h1>
+      <div class="hamburger-menu">
+        <img src="img/icons/hamburger.png" alt="hamburger-menu" />
+      </div>
+    </header>
+    <section class="headline">
+      <h1>I'm Hein Thant Glad to see you!</h1>
```

Comment:
- [ ] Great job on using the right semantic tag for this 🥇 
- [ ] Kindly try to follow the design in Figma as much as possible, please reduce the `margin-top` on this and increase the `font-size` to `40px` 👍